### DISCUSSION TITLE
Why does it install beets in~/.local/bin?

### DISCUSSION THREAD

**Discussion by HubKing** [beetbox/beets#4535] [Q&A] [answered]
It is a Ubuntu Server 22.04. I installed pip and then ran pip install beets, and then ran beet config -p as the doc said, but beet was not found. The document did not give any more information, so I used "find" and found that it is under "~/.local/bin". That path was not in my path by default. 

Why does Beets install the binaries there? Shouldn't it be somewhere where it is in path? Or if not, doesn't the doc at least say that it is installed there so that users would execute ".~/.local/bin/beet cofig -p" instead of "beet config -p"?

**Comment by sampsyo** [comment, marked as answer]
This is a feature of pip:
https://unix.stackexchange.com/questions/712177/defaulting-to-user-installation-because-normal-site-packages-is-not-writeable

You will probably want to extend your `$PATH`, as alluded to here:
https://beets.readthedocs.io/en/stable/faq.html#say-command-not-found

**Comment by JOJ0** [comment]
@HubKing I marked the answer. Did that work out for you?

**Reply by HubKing** [reply]
Yeah, thanks.
